base/doc/clsguide.tex

```diff
@@ -366,7 +366,8 @@ \section{The structure of a class or package}
    declaring new variables, commands and fonts; and loading other files.
 \end{description}
 
-\subsection{Identification}
+\subsection{Identification%
+  \label{sec:ID}}
```

Comment:
Use `Sec` prefix which is used in all but one labels in clsguide.
https://github.com/latex3/latex2e/blob/1f18053be5081bbe8b4a45193d26e101eaeac4f6/base/doc/clsguide.tex#L1085-L1086

```suggestion
\subsection{Identification%
  \label{Sec:ID}}
```
or maybe even move `\label` out of `\subsection`
```suggestion
\subsection{Identification}
\label{Sec:ID}
```

Reply:
actually I would go the other way around  and switch to `sec:` everywhere because that is the typical prefix and clsguide using `Sec:` is anomaly

Reply:
Sorted. The `\label` is in `\section` as we know this is needed in some cases - roll on keyval optional `\section[label = ...]{...}` :)